src/main/java/org/tikv/service/failsafe/HealthCounts.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright 2021 PingCAP, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.tikv.service.failsafe;
+
+public class HealthCounts {
```

Comment:
how about renaming it to `TrafficStats`?

Reply:
sounds bad.

Reply:
`HealthCounts` refes to the implementation of `Hystrix`.